damerau-levenshtein.go

```diff
@@ -15,13 +15,11 @@ func minimum(is ...int) int {
 	return min
 }
 
-var defaultTDL = New(100)
```

Comment:
as a shortcut func anyway using this default value, I think there is no sense to have this global variable. 
And Distance func will be thread safe